### PR TITLE
small fix to printing E_DENSITY_CUBE%TOTAL_DENSITY

### PR DESCRIPTION
**Pull request by Anyborr**
Fixed volume scaling of grid coefficients when printing the electronic density cube with both hard and soft grid contributions

### CODE REVIEW

File: src/qs_scf_post_gpw.F [hunk 1]
```diff
@@ -2160,8 +2160,12 @@ SUBROUTINE write_mo_free_results(qs_env)
             rho_total = rho_hard + rho_soft
             CALL get_pw_grid_info(pw_grid=rho_elec_gspace%pw_grid, &
                                   vol=volume)
+            ! rhotot pw coefficients are by default scaled by grid volume
+            ! need to undo this to get proper charge from printed cube
+            CALL pw_scale(rho_elec_gspace, 1/volume)
```
Comment (oschuett) [inline src/qs_scf_post_gpw.F]: This should probably read `1.0_dp / volume` to ensure a double precision division. Same below.

Comment (Anyborr) [reply src/qs_scf_post_gpw.F]: thank you, fixed it. :)

**Comment by oschuett** [comment]
Could you please run `make pretty`. The other test failures are not related to this PR.

**Comment by mkrack** [comment]
The 3 failing tests are now fixed in the CP2K master. Pull these changes into your local branch (e.g. with `get pull --rebase upstream master`) and perform `make pretty` as already suggested by @oschuett, then `git push` again.

**Pull request merged**